src/avif.c

```diff
@@ -179,44 +179,42 @@ avifResult avifImageCopy(avifImage * dstImage, const avifImage * srcImage, avifP
     avifRWDataSet(&dstImage->exif, srcImage->exif.data, srcImage->exif.size);
     avifImageSetMetadataXMP(dstImage, srcImage->xmp.data, srcImage->xmp.size);
 
+    int firstPlane = AVIF_CHAN_A; // No plane by default.
```

Comment:
this should be stable, but an assert that A > V might clear up the meaning of the comment